torch/csrc/dynamo/eval_frame.c

```diff
@@ -18,6 +18,7 @@ static char compile_context[MAX_COMPILE_CONTEXT_SIZE];
 static int active_dynamo_threads = 0;
 
 static Py_tss_t eval_frame_callback_key = Py_tss_NEEDS_INIT;
+static bool eval_frame_callback_enabled = true;
```

Comment:
Are you sure this doesn't need to be thread local 🤔

Reply:
I think `eval_frame_callback_key ` and `eval_frame_callback_enabled ` storage class specifiers should match since these two vars are used in similar ways. Would not using `thread_local` here lead to bugs right now, or are we future-proofing for multithreaded dynamo?